.github/workflows/035-cargo-test-lint-aurae-builder-make-musl-proto-test-lint.yml

```diff
@@ -30,43 +30,43 @@
 #
 name: "Cargo Test Lint (035) [aurae-builder:builder-latest]"
 on:
- push:
-   branches: main
- pull_request:
-   branches: main
+  push:
+    branches: main
+  pull_request:
+    branches: main
 
 permissions:
- contents: read
- packages: write
+  contents: read
+  packages: write
 env:
- CARGO_TERM_COLOR: always
+  CARGO_TERM_COLOR: always
 jobs:
- build-container:
-   uses: ./.github/workflows/030-ubuntu-aurae-builder-image-ubuntu-docker-build-builder.yml
- build:
-   name: Build (lint, compile, test)
-   runs-on: ubuntu-latest
-   needs: build-container
-   timeout-minutes: 20
-   container:
-     image: ghcr.io/${{ github.repository }}/aurae-builder:builder-latest
-     credentials:
-       username:  ${{ github.actor }}
-       password:  ${{ secrets.GITHUB_TOKEN }}
-   steps:
-     - uses: actions/checkout@v3
-     - uses: actions/cache@v3
-       with:
+  build-container:
+    uses: ./.github/workflows/030-ubuntu-aurae-builder-image-ubuntu-docker-build-builder.yml
+  build:
+    name: Build (lint, compile, test)
+    runs-on: ubuntu-latest
+    needs: build-container
+    timeout-minutes: 20
+    container:
+      image: ghcr.io/${{ github.repository }}/aurae-builder:builder-latest
+      credentials:
+        username: ${{ github.actor }}
+        password: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/cache@v3
+        with:
           path: |
             ~/.cargo/bin/
             ~/.cargo/registry/index/
             ~/.cargo/registry/cache/
             ~/.cargo/git/db/
             target/
           key: build-compile-test-container-${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
-     - name: Cargo Test Lint [make test lint]
-       # This should remain the only command we execute as this matches the title of the file.
-       # The goal is for this to be easy to find from the GitHub dashboard.
-       # Instead of adding more commands here, consider another make target or a new YAML file
-       # named with a good name.
-       run: make musl proto test lint
+      - name: Cargo Test Lint [make test lint]
+        # This should remain the only command we execute as this matches the title of the file.
+        # The goal is for this to be easy to find from the GitHub dashboard.
+        # Instead of adding more commands here, consider another make target or a new YAML file
+        # named with a good name.
+        run: make musl lint test
```

Comment:
the file should be renamed if we're dropping proto.  i do wonder why this isn't just "make test" and assume the dependent jobs are correctly run.  i think this would be safer, no?

Reply:
I like that idea. I'll add lint to test